src/video_core/renderer_vulkan/liverpool_to_vk.cpp

```diff
@@ -223,6 +223,7 @@ vk::CompareOp DepthCompare(AmdGpu::DepthCompare comp) {
     case AmdGpu::DepthCompare::Always:
         return vk::CompareOp::eAlways;
     default:
+        UNREACHABLE_MSG("Unknown depth compare op {}", u32(comp));
```

Comment:
Why is this done twice? if your adding a new one the old one should be removed

Reply:
Someone updated the branch and I just synced it.